test/test_environment.rb

```diff
@@ -746,6 +746,19 @@ def foo; end
     @env.unregister_preprocessor('application/javascript', processor)
     assert_equal "// =require \"notfound\"\n;\n", @env["missing_require.js"].to_s
   end
+
+  test "disabling processors by class name" do
```

Comment:
Can you update the test name to `disabling processors by class name also disables processors which are instances of that class` to make it more clear?